.dlv/config.yml

```diff
@@ -0,0 +1,54 @@
+# Configuration file for the delve debugger.
```

Comment:
Please remove the file, it's for debugging.

Reply:
done